src/Umbraco.Community.Contentment/DataEditors/ImageButtonList/image-button-list.html

```diff
@@ -0,0 +1,22 @@
+﻿<!-- Copyright © 2013-present Umbraco.
+   - Parts of this Source Code has been derived from Umbraco CMS.
+   - https://github.com/umbraco/Umbraco-CMS/blob/release-8.1.5/src/Umbraco.Web.UI.Client/src/views/components/forms/umb-radiobutton.html
+   - Modified under the permissions of the MIT License.
+   - Modifications are licensed under the Mozilla Public License.
+   - Copyright © 2019 Lee Kelleher and Marc Stöcker.
+   - This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at https://mozilla.org/MPL/2.0/. -->
+
+<div class="contentment umb-imagebuttons" ng-class="vm.uniqueId" ng-controller="Umbraco.Community.Contentment.DataEditors.ImageButtonList.Controller as vm">
+    <ul class="unstyled radio no-indent image-selector">
+        <li ng-repeat="item in vm.items">
+            <input type="radio" value="{{item.value}}" ng-model="model.value" ng-disabled="item.disabled" id="{{vm.uniqueId + '_' + item.value}}" />
+            <label for="{{vm.uniqueId + '_' + item.value}}" class="sprite-image" ng-class="{ 'umb-form-check--disabled': item.disabled }" title="{{item.name}}">
+                <svg class="imagebutton-icon">
+                    <use ng-href="{{vm.svgSpriteUrl}}#{{item.value}}" xlink:href="" />
+                </svg>
```

Comment:
_A note for myself._ 🤫

Boiling it down, this is the crux of the UX. _In this case, the SVG fragment is for displaying the image/value/intention._ I wonder if this markup was templatable, could this satisfy other custom experiences?  Could we be looking at a "Templated List" UI? a la https://github.com/uComponents/nuPickers/wiki/Custom-Labels // although the use of Macros feels quite heavy for this - I'd probably be thinking AngularJS views.

Reply:
Follow up on this... I've prototyped a "Templated List" editor, that accepts an AngularJS template.

![Annotation 2020-05-05 141023](https://user-images.githubusercontent.com/209066/81069926-7aa6fc00-8eda-11ea-81d5-c329c110e9f9.png)

It's working pretty well.

@MarcStoecker I wonder if a Templated List editor would work for you, with regards to rendering SVG fragments?
_(Note, this is only prototyped on my machine, I haven't committed anything to the repo yet)_

Reply:
Yes, I would think so. Happy to try it out as soon as you push something! 🙃